src/state/actions/decks.js

```diff
@@ -86,6 +87,19 @@ export const deleteDeck = id => async dispatch => {
   }
 };
 
+export const createDeck = payload => async dispatch => {
+  try {
+    const response = await axiosWithAuth().post(`/decks`, payload);
+    dispatch({
+      type: CREATE_DECK,
+      payload: payload,
+    });
+    return response;
+  } catch (err) {
+    throw err;
+  }
+};
+
```

Comment:
1. I'm not sure what is being dispatched here, because you are not dispatching the response from axios call.
2. On line 95 you could use shorthand for property creation.

Reply:
The deck that was created by the user